src/kep/validation/checkpoints.py

```diff
@@ -211,16 +219,19 @@ def as_checkpoints(text):
     m=as_checkpoints(MONTHLY_STR)
 )
 
+OPTIONAL_CHECKPOINTS = dict(
+    a=as_checkpoints(ANNUAL_STR_2016),
+    q=[],
```

Comment:
Stub values to avoid awkward calls to this method.

Reply:
This is also seems as a valid inline comment in code